seagull.docker

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.3
+MAINTAINER tobe "tobeg3oogle@gmail.com"
+MAINTAINER Florin Patan "florinpatan@gmail.com"
+
+ARG BINARY_FILE
+
+# DNS stuff
+RUN echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf
```

Comment:
Yes, I've seen many recommendation for it. Not sure if it still appllies with the Go only dns resolver.